.github/workflows/steam.yml

```diff
@@ -68,24 +68,24 @@ jobs:
           --mount type=bind,ro,src=$(pwd)/get-pip.py,dst=/endless-sky/get-pip.py \
           -e SCHROOT_CHROOT_NAME=steamrt_scout_${{ matrix.platform }} \
           -e LTO_PLUGIN_PATH=/usr/lib/gcc/${{ matrix.arch }}-linux-gnu/5/liblto_plugin.so \
-          -w /endless-sky s_rt /bin/bash -c \
-          'sudo dpkg --clear-avail \
-            && echo "deb http://old-releases.ubuntu.com/ubuntu/ precise main restricted universe multiverse" > /etc/apt/sources.list \
-            && echo "deb http://old-releases.ubuntu.com/ubuntu/ precise-updates main restricted universe multiverse" >> /etc/apt/sources.list \
-            && echo "deb http://old-releases.ubuntu.com/ubuntu/ precise-security main restricted universe multiverse" >> /etc/apt/sources.list \
-            && sudo apt-get update -q \
-            && sudo apt-get install -yq --no-install-recommends libmad0-dev \
-            && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-5 101 \
-            && update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-5 101 \
-            && update-alternatives --install /usr/bin/cpp cpp-bin /usr/bin/cpp-5 101 \
-            && update-alternatives --auto gcc \
-            && update-alternatives --auto g++ \
-            && update-alternatives --auto cpp-bin \
-            && python3.5 get-pip.py \
-            && python3.5 -m pip install SCons==4.2.0 \
-            && python3.5 $(which scons) -Qj $(nproc) endless-sky test \
-            && mv ./endless-sky build/ \
-            && ./build/endless-sky -v \
+          -w /endless-sky s_rt /bin/bash -o pipefail -exc \
+          'sudo dpkg --clear-avail; \
```

Comment:
If `\` were to be removed you could even remove the semicolons and this would look cleaner and still work as intended.